src/_includes/layouts/base.njk

```diff
@@ -0,0 +1,165 @@
+        {% endif %}
+    </head>
+    <body class="infusion-docs fl-theme-prefsEditor-default">
+        <div class="container infusion-docs-container">
+
+            <!-- Skip Link -->
+            <div id="skip">
+                <a href="#tableOfContents">Jump to Table of Contents</a>
+                <a href="#content">Skip to Content</a>
+            </div>
+            <!-- END skip link -->
+
+            <!-- BEGIN markup for Preference Editor -->
+            <div class="flc-prefsEditor-separatedPanel fl-prefsEditor-separatedPanel">
+                <!-- This is the div that will contain the Preference Editor component -->
+                <div class="flc-slidingPanel-panel flc-prefsEditor-iframe"></div>
+
+                <!-- This div is for the sliding panel that shows and hides the Preference Editor controls -->
+                <div class="fl-panelBar">
+                    <span class="fl-prefsEditor-buttons">
+                        <button id="reset" class="flc-prefsEditor-reset fl-prefsEditor-reset"><span class="fl-icon-undo"></span> Reset</button>
+                        <button id="show-hide" class="flc-slidingPanel-toggleButton fl-prefsEditor-showHide">+ Show Preferences</button>
+                    </span>
+                </div>
+            </div>
+
+            <script type="text/javascript">
+                fluid.uiOptions.prefsEditor(".flc-prefsEditor-separatedPanel", {
+                    terms: {
+                        "templatePrefix": "{{ "/lib/infusion/src/framework/preferences/html" | url }}",
+                        "messagePrefix": "{{ "/lib/infusion/src/framework/preferences/messages" | url }}"
+                    },
+                    "tocTemplate": "{{ "/lib/infusion/src/components/tableOfContents/html/TableOfContents.html" | url }}"
+                });
+            </script>
+
+            <!-- END markup for Preference Editor -->
+
+            <header class="row infusion-docs-header">
+                <div class="infusion-docs-fluidLogo small-12 medium-3 column">
+                    <a class="infusion-docs-fluidLogoText" href="{{ "/" | url }}" title="Infusion">infusion</a>
+                </div>
+
+                <nav class="infusion-docs-fluidCategories small-12 medium-9 column">
+                    {% set navItems = collections.all | eleventyNavigation %}
+                    {% for item in navItems %}
+                        <a href="{{ item.url }}"{% if page.url === item.url %} aria-current="page"{% endif %}{% if item.key === category%} class="infusion-docs-category-active"{% endif %}>{{ item.key }}</a>
+                    {% endfor %}
+                </nav>
+            </header>
+
+            <div class="infusion-docs-mainBody">
+                <nav id="tableOfContents" class="infusion-docs-TOC small-12 medium-3 column" tabindex="-1">
+                    {% if category === "Search" %}
+                    {% else %}
```

Comment:
Rather than having an empty `if` block, shall these two lines be replaced with `{% if category !== "Search" %}`?